test/test.js

```diff
@@ -251,6 +251,28 @@ describe('HttpsProxyAgent', function() {
 
 			req.on('abort', done);
 		});
+		it('should emit an "end" event on the `http.IncomingMessage` if the proxy responds with non-200 status code', function(done) {
```

Comment:
I would like like to shorten this title but I can't think of a more concise way to say the same thing.